flags.go

```diff
@@ -194,17 +207,17 @@ func (f *FlagClause) formatPlaceHolder() string {
 	if f.placeholder != "" {
 		return f.placeholder
 	}
-	if f.defaultValue != "" {
+	if len(f.defaultValues) > 0 {
 		if _, ok := f.value.(*stringValue); ok {
-			return fmt.Sprintf("%q", f.defaultValue)
+			return fmt.Sprintf("%q", f.defaultValues[0])
```

Comment:
this doesn't play very nice with multiple default values, but listing them all would break the current format, so I thought this was "enough"